en/core-libraries/caching.rst

```diff
@@ -232,7 +232,7 @@ newer, using the ``remember()`` method would look like::
 
         public function newest() {
             $model = $this;
-            return Cache::remember('newest_posts', function() use ($model){
+            return Cache::remember('newest_posts', function () use ($model){
```

Comment:
use ($model){

=> 

use ($model) {

If you apply this best to do it completely.